Got an error when running `pytest` in Debug mode

Hello,

I got the following error when I run the test on a fresh `cookiecutter` instance with `FLASK_DEBUG=True`:
```
    def init_config(self, app):
        """Initialize configuration.

            :param app: An instance of :class:`~flask.Flask`.
            """
        config_apps = ['APP_', 'RATELIMIT_']
        flask_talisman_debug_mode = ["'unsafe-inline'"]
        for k in dir(config):
            if any([k.startswith(prefix) for prefix in config_apps]):
                app.config.setdefault(k, getattr(config, k))
        if app.config['DEBUG']:
            app.config['APP_DEFAULT_SECURE_HEADERS'][
>               'content_security_policy']['default-src'] += \
                flask_talisman_debug_mode
E           KeyError: 'content_security_policy'

../invenio-app/invenio_app/ext.py:65: KeyError
```

--- a/invenio_app/ext.py
+++ b/invenio_app/ext.py
@@ -61,6 +61,13 @@ class InvenioApp(object):
             if any([k.startswith(prefix) for prefix in config_apps]):
                 app.config.setdefault(k, getattr(config, k))
         if app.config['DEBUG']:
+            # set defaults if config overridden
+            app.config.setdefault('APP_DEFAULT_SECURE_HEADERS', {})
+            app.config['APP_DEFAULT_SECURE_HEADERS'].setdefault(
+                'content_security_policy', {})
+            app.config['APP_DEFAULT_SECURE_HEADERS'][
+                'content_security_policy'].setdefault('default-src', [])
+            # add default csp value when debug
             app.config['APP_DEFAULT_SECURE_HEADERS'][
                 'content_security_policy']['default-src'] += \
                 flask_talisman_debug_mode

--- a/tests/test_app.py
+++ b/tests/test_app.py
@@ -36,8 +36,8 @@ def test_headers(app):
     def jessica_jones():
         return 'jessica jones'
 
-    @app.route('/avangers')
-    def avangers():
+    @app.route('/avengers')
+    def avengers():
         return 'infinity war'
 
     with app.test_client() as client:
@@ -53,7 +53,7 @@ def test_headers(app):
         assert res.headers['X-RateLimit-Remaining']
         assert res.headers['X-RateLimit-Reset']
 
-        res = client.get('/avangers')
+        res = client.get('/avengers')
         assert res.status_code == 200
         assert res.headers['X-Content-Security-Policy']
         assert res.headers['X-Content-Type-Options']
@@ -62,3 +62,31 @@ def test_headers(app):
         assert res.headers['X-RateLimit-Limit']
         assert res.headers['X-RateLimit-Remaining']
         assert res.headers['X-RateLimit-Reset']
+
+
+def _test_csp_default_src(app, expect):
+    """Assert that the Content-Security-Policy header is the expect param."""
+    ext = InvenioApp(app)
+
+    @app.route('/captain_america')
+    def captain_america():
+        return 'captain america'
+
+    with app.test_client() as client:
+        res = client.get('/captain_america')
+        assert res.status_code == 200
+        assert res.headers['Content-Security-Policy'] == expect
+        assert res.headers['X-Content-Security-Policy'] == expect
+
+
+def test_csp_default_src_when_debug_false(app):
+    """Test the Content-Security-Policy header when app debug is False."""
+    expect = "default-src 'self'"
+    _test_csp_default_src(app, expect)
+
+
+def test_csp_default_src_when_debug_true(app):
+    """Test the Content-Security-Policy header when app debug is True."""
+    app.config['DEBUG'] = True
+    expect = "default-src 'unsafe-inline'"
+    _test_csp_default_src(app, expect)

